spec/fixtures/files/searchable/test_results_controller.yaml

```diff
@@ -294,3 +294,33 @@
         :account: ${account}
         :policy_id: ${policy.id}
   :query: (group_name ^ "one,two")
+
+- :name: "in search by failed_rule_severity"
+  :entities:
+    :found:
+      - :factory: :v2_test_result
+        :account: ${account}
+        :policy_id: ${policy.id}
+        :additional_rule_results:
+          - severity: 'high'
+            result: 'fail'
```

Comment:
Please add more than just a single rule result here + combine them with passing tests with higher severity to properly cover the DB lookup.